res/sections/28-Les23052017.tex

```diff
@@ -1,81 +1,40 @@
+\subsubsection{IS Auditor e IT Governance}
 
-\chapter{Security Administrator}
-\label{SA}
-
-\section{Security Operations}
-\label{SA:SO}
-
-Operazioni relative alla sicurezza:
-\begin{itemize}
-\item Gestione delle identità e controllo degli accessi: per esempio badge 
-all'ingresso ecc. Ma non solo! Anche controlli di tipo informatico devono
-essere aggiornati dopo ad esempio il licenziamento di un impiegato, al quale
-dev'essere negato l'accesso ai dati aziendali alla fine del suo ultimo giorno
-lavorativo;
-\item \textit{Patching} del sistema e gestione delle configurazioni: il
-patching è l'anno zero della sicurezza;
-\item Controlli dei cambiamenti e gestione dei rilasci: Il controllo dei 
-cambiamenti ogni cambiamento deve essere documentato e i test sono stati 
-eseguiti allora può essere introdotto.
-I rilasci software devono essere prima di tutto testati e documentati, e poi
-devono essere autorizzati dall'apposito responsabile. Questo procedimento
-garantisce che il software introdotto nel sistema non causi anomalie;
-\item Metriche per la sicurezza e reporting: Ogni società sa dove bisogna 
-andare. Una tipica metrica di sicurezza in caso di \emph{awareness} interna,
-consiste nel mandare false email di spam ai dipendenti per vedere se
-hanno recepito le lezioni sulla sicurezza;
-\item Mantenimento dei controlli tecnologici: i controlli vanno mantenuti. 
-Il controllo di sicurezza è un qualcosa che da un risparmio in termini di costi 
-se applicata in maniera efficace. Vendere la sicurezza è difficile, e un buon 
-manager è in grado di venderla;
-
-\item Risposta all'incidente, investigazione e conseguente risoluzione
-Il problema non è come si viene attaccati ma quando si è attaccati: bisogna 
-avere un buon grado di comunicazione e trovare la causa dell'incidente. Anche 
-avere una buona soluzione, per prevenire che l'incidente si verifichi di nuovo.
-\textit{Investigation} è la fase successiva all'attacco e serve per capire 
-perché e come è successo quello che è successo.
-
-\end{itemize}
-
-
-\subsection{IS Auditor e IT Governance}
-
-Esistono cinque indicatori per capire se l'azienda ha fatto un buon lavoro nel 
-veicolare i propri obiettivi ai lavoratori. È importante che i dipendenti 
-sappiano dare risposte a 
+Esistono cinque indicatori per capire se l'azienda ha fatto un buon lavoro nel
+veicolare i propri obiettivi ai lavoratori. È importante che i dipendenti
+sappiano dare risposte a
 \begin{enumerate*}[label=\arabic*)]
-\item cosa è allineata la compagnia, 
-\item qual è la sua missione, 
-\item visione, 
-\item valori, 
+\item cosa è allineata la compagnia,
```

Comment:
Secondo me è sbagliata anche cosi perchè quella `a` me la vedo in sostituzione a `alle domande` e la comanda è `A cosa è allineata l'azienda?` e non `Cosa è allineata l'azienda?` da quello che ho capito (più che altro la seconda non ha senso:sweat_smile:)